packages/slate/src/interfaces/editor.ts

```diff
@@ -1575,6 +1575,6 @@ export const Editor: EditorInterface = {
  * A helper type for narrowing matched nodes with a predicate.
  */
 
-type NodeMatch<T extends Node> =
-  | ((node: Node) => node is T)
-  | ((node: Node) => boolean)
+export type NodeMatch<T extends Node> =
```

Comment:
Will this type break existing implementations that have a `NodeMatch` w/o a `path` parameter?  Does everyone now need to update their `match` functions?  I think no but figured I'd double-check...

Reply:
It shouldn't, adding a second possible parameter like this does not necessarily mean you have to pass a second argument every time. Like when you call a `forEach`, you can call it with the element and index or just the element.

But if you really want me to double check, I can. Is there a simple way to make this version of the code run in my own project?

Reply:
No it should be fine there's ton's of `match` functions in Slate core with just `node` so TS would've thrown an error in Slate core is there was an issue..